scripts/lint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
where does this file get used?

Reply:
Just manually, documented in `docs/contribute.md`. I can put the command directly in the doc instead if preferred.

Reply:
This makes me wondering should we just use the `dotnet-format` and ignore the `super-linter` altogether or the reverse way.
Want to make sure the `linter workflow` and the `dev.sh format` lines up with each other.

Reply:
Under the hood, `super-linter` uses `dotnet-format`. It'll also read the `./src/.editorconfig` settings the same as `dotnet-format`.

 Being new to C#, I couldn't find any high usage GitHub Actions on the marketplace for `dotnet-format` specific linting. If there's one you're familiar with that'll support both, let me know.

However, if we also wanted to enforce any lints on shell, cmd, pwsh, md, etc. files that also are in here, then super-linter supports all of them.